src/main/java/org/apache/tools/bzip2/CBZip2InputStream.java

```diff
@@ -29,957 +29,918 @@
+                }
+            } else {
+                this.su_rNToGo--;
+            }
+            this.su_j2 = 0;
+            this.currentState = RAND_PART_C_STATE;
+            if (this.su_rNToGo == 1) {
+                this.su_z ^= 1;
+            }
+            setupRandPartC();
+        } else {
+            this.currentState = RAND_PART_A_STATE;
+            setupRandPartA();
+        }
+    }
+
+    private void setupRandPartC() throws IOException {
+        if (this.su_j2 < this.su_z) {
+            this.currentChar = this.su_ch2;
+            this.crc.updateCRC(this.su_ch2);
+            this.su_j2++;
+        } else {
+            this.currentState = RAND_PART_A_STATE;
+            this.su_i2++;
+            this.su_count = 0;
+            setupRandPartA();
+        }
+    }
+
+    private void setupNoRandPartB() throws IOException {
+        if (this.su_ch2 != this.su_chPrev) {
+            this.su_count = 1;
+            setupNoRandPartA();
+        } else if (++this.su_count >= 4) {
+            this.su_z = (char) (this.data.ll8[this.su_tPos] & 0xff);
+            this.su_tPos = this.data.tt[this.su_tPos];
+            this.su_j2 = 0;
+            setupNoRandPartC();
+        } else {
+            setupNoRandPartA();
+        }
+    }
+
+    private void setupNoRandPartC() throws IOException {
+        if (this.su_j2 < this.su_z) {
+            int su_ch2Shadow = this.su_ch2;
+            this.currentChar = su_ch2Shadow;
+            this.crc.updateCRC(su_ch2Shadow);
+            this.su_j2++;
+            this.currentState = NO_RAND_PART_C_STATE;
+        } else {
+            this.su_i2++;
+            this.su_count = 0;
+            setupNoRandPartA();
+        }
+    }
+
+    private static final class Data extends Object {
+
+        // (with blockSize 900k)
+        final boolean[] inUse = new boolean[256]; //      256 byte
+
+        final byte[] seqToUnseq = new byte[256]; //      256 byte
+        final byte[] selector = new byte[MAX_SELECTORS]; //    18002 byte
+        final byte[] selectorMtf = new byte[MAX_SELECTORS]; //    18002 byte
+
+        /**
+         * Freq table collected to save a pass over the data during
+         * decompression.
+         */
+        final int[] unzftab = new int[256]; //     1024 byte
+
+        final int[][] limit = new int[N_GROUPS][MAX_ALPHA_SIZE]; //     6192 byte
+        final int[][] base = new int[N_GROUPS][MAX_ALPHA_SIZE]; //     6192 byte
+        final int[][] perm = new int[N_GROUPS][MAX_ALPHA_SIZE]; //     6192 byte
+        final int[] minLens = new int[N_GROUPS]; //       24 byte
+
+        final int[] cftab = new int[257]; //     1028 byte
+        final char[] getAndMoveToFrontDecode_yy = new char[256]; //      512 byte
+        final char[][] temp_charArray2d = new char[N_GROUPS][MAX_ALPHA_SIZE]; //     3096 byte
+        final byte[] recvDecodingTables_pos = new byte[N_GROUPS]; //        6 byte
+        //---------------
+        //    60798 byte
+
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS00117)